sklearn/decomposition/tests/test_dict_learning.py

```diff
@@ -540,18 +540,6 @@ def test_sparse_encode_positivity(algo, positive):
         assert (code < 0).any()
 
 
-@pytest.mark.parametrize("algo", ["lars", "omp"])
```

Comment:
This can actually stay, since it tests the rejection of the `lars, omp` algorithms only when `positive=True`, hence it is conditional validation not simple validation

Reply:
I see, thks! I ll put it back in. On the other side, I don't see other tests that can be removed, besides the one you pointed out earlier.